aggregator/elem.go

```diff
@@ -728,3 +691,12 @@ func (e *GaugeElem) processValue(timeNanos int64, agg aggregation.Gauge, fn aggM
 		fn(e.opts.FullGaugePrefix(), e.id, e.suffix(aggType), timeNanos, agg.ValueOf(aggType), e.sp)
 	}
 }
+
+// NB(cw) suffix overrides default suffixes to not add
+// suffix for default Gauge metrics for backward compatibility reasons.
+func (e *GaugeElem) suffix(aggType policy.AggregationType) []byte {
+	if aggType == policy.Last {
```

Comment:
Should we add `defaultGaugeAggregationType` to the options and compare against it?